egs/ami/s5b/RESULTS_mdm

```diff
@@ -86,6 +86,13 @@
 
 #  local/chain/tuning/run_tdnn_lstm_1i.sh --mic mdm8 --use-ihm-ali true --train-set train_cleaned  --gmm tri3_cleaned
 #  cleanup + chain TDNN+LSTM model, MDM audio and alignments from IHM data
-# *** best system ***
 %WER 34.6 | 15116 94508 | 69.6 17.6 12.9 4.1 34.6 62.3 | 0.687 | exp/mdm8/chain_cleaned/tdnn_lstm1i_sp_bi_ihmali_ld5/decode_dev/ascore_9/dev_hires_o4.ctm.filt.sys
 %WER 37.1 | 14343 90002 | 66.3 18.8 14.9 3.4 37.1 62.3 | 0.659 | exp/mdm8/chain_cleaned/tdnn_lstm1i_sp_bi_ihmali_ld5/decode_eval/ascore_9/eval_hires_o4.ctm.filt.sys
+
+
+# local/chain/multi_condition/run_tdnn_lstm.sh --mic mdm8 --use-ihm-ali true --train-set train_cleaned --gmm tri3_cleaned
+# cleanup + chain TDNN+LSTM model, MDM original + IHM reverberated data, alignments from IHM data
+# *** best system ***
+%WER 31.8 | 14488 94497 | 71.8 15.4 12.8 3.5 31.8 62.7 | 0.698 | exp/mdm8/chain_cleaned_rvb/tdnn_lstm1i_sp_rvb_bi_ihmali/decode_dev/ascore_10/dev_hires_o4.ctm.filt.sys
```

Comment:
@tomkocse Very impressive results. Please add the version number of your script which generates these results (eg. run_tdnn_lstm_1a.sh ) as we might replace run_tdnn_lstm.sh with a better tdnn_lstm architecture in the future.

Move this script to tuning directory as run_tdnn_lstm_1b.sh and softlink it from run_tdnn_lstm.sh